src/pages/Homepage/LendByCategoryHomepage.vue

```diff
@@ -157,6 +157,47 @@
 				<h2 class="lender-quotes__header text-center small-12 columns">
 					What our lending <br class="so mo"> community thinks
 				</h2>
+				<div
+					class="lender-quotes__lender-quote--card small-10 medium-8 large-5 xlarge-4 columns"
```

Comment:
I would recommend making the cards themselves their own BEM "block" since they could stand alone easily without the lender-quotes section.. Maybe .quote-card or something.